pkg/diagnose/scan.go

```diff
@@ -8,8 +8,8 @@ import (
 	"runtime"
 	"time"
 
-	nvidia_component_error_sxid "github.com/leptonai/gpud/components/accelerator/nvidia/error/sxid"
-	nvidia_component_error_xid "github.com/leptonai/gpud/components/accelerator/nvidia/error/xid"
+	error_sxid "github.com/leptonai/gpud/components/accelerator/nvidia/error/sxid"
+	error_xid "github.com/leptonai/gpud/components/accelerator/nvidia/error/xid"
```

Comment:
These two aliases are confusing and can easily mislead us into thinking that these two packages define error types.

Reply:
renamed to `nvidia_x/sxid`